test/protocol/variables_test.rb

```diff
@@ -0,0 +1,85 @@
+# frozen_string_literal: true
+
+require_relative "../support/protocol_test_case"
+
+module DEBUGGER__
+  class DAPVariablesTest < ProtocolTestCase
```

Comment:
Nitpick: since the program script for this class only has global variables, could we be more specific on the class name, like `DAPGlobalVariablesTest`?